bin/seagull.js

```diff
@@ -32,8 +34,12 @@ var defaults = {
   contentDir : 'content',
   fileDir    : 'files',
   htmlDir    : 'html',
+  pretty     : false
```

Comment:
Moved `pretty` here and defaulted to `false` instead of the 3 `cfg.pretty || false` references previously.

Like I say, this is a kitchen-sink PR of epic unrelated-ness.